restli-server/src/main/java/com/linkedin/restli/server/config/ResourceMethodConfigImpl.java

```diff
@@ -10,15 +10,17 @@
  *
  * @author mnchen
  */
-class ResourceMethodConfigImpl implements ResourceMethodConfig
+public class ResourceMethodConfigImpl implements ResourceMethodConfig
```

Comment:
My last question before giving a ship:

Why is this public? Is this used anywhere?

Reply:
this is need for RoutingResult default initialization.

Reply:
sorry I still didn't quite get why package private is not enough.
Aren't they in the same package? RoutingResult  is in the parent package of ResourceMethodConfigImpl ?

Reply:
routingresult is not in same package